crates/cli/src/opts/dependency.rs

```diff
@@ -360,4 +370,26 @@ mod tests {
         let dep: Dependency = "org-git12345678@github.com:my-org/my-repo.git".parse().unwrap();
         assert_eq!(dep.url.unwrap(), "https://github.com/my-org/my-repo");
     }
+
+    #[test]
+    fn can_parse_with_explicit_ref_type() {
+        let dep = Dependency::from_str("smartcontractkit/ccip@tag=contracts-ccip/v1.2.1").unwrap();
+        assert_eq!(dep.name, "ccip");
+        assert_eq!(dep.url, Some("https://github.com/smartcontractkit/ccip".to_string()));
+        assert_eq!(dep.tag, Some("contracts-ccip/v1.2.1".to_string()));
+        assert_eq!(dep.alias, None);
+
+        let dep =
+            Dependency::from_str("smartcontractkit/ccip@branch=contracts-ccip/v1.2.1").unwrap();
+        assert_eq!(dep.name, "ccip");
+        assert_eq!(dep.url, Some("https://github.com/smartcontractkit/ccip".to_string()));
+        assert_eq!(dep.tag, Some("contracts-ccip/v1.2.1".to_string()));
+        assert_eq!(dep.alias, None);
+
+        let dep = Dependency::from_str("smartcontractkit/ccip@commit=80eb41b").unwrap();
```

Comment:
Soldeer / Rust uses `rev` to refer to a commit, perhaps it makes sense to align here

Reply:
changed in https://github.com/foundry-rs/foundry/pull/9214/commits/e9d2967e943ac6a2fa5a0856db5d95e1b7145231, please check